exercises/structs/structs3.rs

```diff
@@ -37,31 +37,31 @@ mod tests {
     #[test]
     #[should_panic]
     fn fail_creating_weightless_package() {
-        let country_from = String::from("Spain");
-        let country_to = String::from("Austria");
+        let sender_country = String::from("Spain");
+        let recipient_country = String::from("Austria");
 
-        Package::new(country_from, country_to, -2.21);
+        Package::new(sender_country, recipient_country, -2210);
     }
 
     #[test]
     fn create_international_package() {
-        let country_from = String::from("Spain");
-        let country_to = String::from("Russia");
+        let sender_country = String::from("Spain");
+        let recipient_country = String::from("Russia");
         
-        let package = Package::new(country_from, country_to, 1.2);
+        let package = Package::new(sender_country, recipient_country, 1200);
 
         assert!(package.is_international());
     }
 
     #[test]
     fn calculate_transport_fees() {
-        let country_from = String::from("Spain");
-        let country_to = String::from("Spain");
+        let sender_country = String::from("Spain");
+        let recipient_country = String::from("Spain");
 
-        let country_fee = ???;
+        let cents_per_kg = ???;
```

Comment:
Random thought, totally out of scope for this PR: it feels weird to me that we have learners guess backwards the appropriate input to obtain a predetermined output.

Also, I'm starting to think that forcing miscompilations through invalid code like `???` is something we should dial back a bit.